src/components/WeekDayCard.js

```diff
@@ -0,0 +1,48 @@
+import "./WeekDayCard.css";
+import { HiSun } from "react-icons/hi";
+import { FaMoon } from "react-icons/fa";
+
+export default function WeekDayCard() {
+  return (
+    <div className="WeekDayCard">
+      <div className="weekday">
+        <h4 className="weekday-headline">monday</h4>
```

Comment:
the font-colour is black on my screen right now. How does it look if you do the same colour as on the header?